jenkins/bootstrap.py

```diff
@@ -729,15 +729,20 @@ def job_script(job, use_json):
     """Return path to script for job."""
     if not use_json:
         return [test_infra('jobs/%s.sh' % job)]
+
     with open(test_infra('jobs/config.json')) as fp:
         config = json.loads(fp.read())
+    job_config = config.get(job)
+
+    if job_config is None:
+        with open(test_infra('jobs/generated/config.json')) as fp:
+            config = json.loads(fp.read())
```

Comment:
I will just add a `"tag": ["generated"]` field into the generated job definition. Each run of the tool would remove all configs with this tag, regenerate the new ones and append them at the end.